pkg/statsd/handler_http_forwarder_v2.go

```diff
@@ -299,6 +302,8 @@ func (hfh *HttpForwarderHandlerV2) emitMetrics(statser stats.Statser) {
 	statser.Report("http.forwarder.sent", &hfh.messagesSent, nil)
 	statser.Report("http.forwarder.retried", &hfh.messagesRetried, nil)
 	statser.Report("http.forwarder.dropped", &hfh.messagesDropped, nil)
+	statser.TimingMS("http.forwarder.post_latency.max", float64(hfh.postLatencyMax.Load()), nil)
```

Comment:
This is a tricky decision on what to emit here.  There's going to be several values sent in the aggregator flush interval (remember the forwarder flushes a lot faster than the aggregator).

Right now, we're getting the lifetime maximum, since we don't reset the max.  That may be acceptable?

If we reset the max:
- If we emit a timer, then we're back in the same boat of "all the cardinality".
- If we emit a gauge, then only the last forwarder flush interval will be recorded.

If we don't reset the max:
- If we emit a timer, then we're back in the same boat of "all the cardinality".
- If we emit a gauge, then we get the process lifetime maximum (which may be fine)